### DISCUSSION TITLE
can it work for appletv

### DISCUSSION THREAD

**Discussion by aboutimes** [MonitorControl/MonitorControl#1492] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Discussion

some tv's hdmi cec not surport to appletv ,but MonitorControl work for it on macos,can it possibly work for appletv?

**Comment by waydabber** [comment, marked as answer]
The app won't have an AppleTV version I am afraid. HDMI-CEC support is not planned either (although there are some HDMI dongles that might have support for it).
